hy/importer.py

```diff
@@ -235,3 +251,47 @@ def find_module(self, fullname, path=None):
 
 sys.meta_path.insert(0, MetaImporter())
 sys.path.insert(0, "")
+
+
+def is_package(module_name):
+    mpath = os.path.join(*module_name.split("."))
+    for path in map(os.path.abspath, sys.path):
+        if os.path.exists(os.path.join(path, mpath, "__init__.hy")):
+            return True
+    return False
```

Comment:
Evidently, everything still works, since `hy.core` is such a package. I'm not sure why, but I don't really know what `module.__package__` does, anyway.